event-handler/cli.go

```diff
@@ -62,6 +65,45 @@ type TeleportConfig struct {
 	TeleportKey string `help:"Teleport TLS key file" type:"existingfile" env:"FDFWD_TELEPORT_KEY"`
 }
 
+// Validate performs some basic validation on TeleportConfig that can't be handled using the kong defaulting
+func (cfg *TeleportConfig) Validate() error {
```

Comment:
Nit: To make it consistent with how we usually name other methods that validate config structs in our codebase.
```suggestion
func (cfg *TeleportConfig) Check() error {
```